.env.production

```diff
@@ -1,2 +1 @@
-REACT_APP_TEST=test
-REACT_APP_BASE_API_URL=https://test.domain.com/api
```

Comment:
Delete these two variables because they are not used throughout the project, and they are not intended to be used.